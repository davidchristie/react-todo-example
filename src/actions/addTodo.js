export const ADD_TODO = 'ADD_TODO'

export default newTodo => ({
  payload: newTodo,
  type: ADD_TODO
})
