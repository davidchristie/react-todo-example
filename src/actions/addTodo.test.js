import addTodo from './addTodo'

it('matches snapshot', () => {
  const newItem = {
    completed: false,
    text: 'Test todo item test'
  }
  expect(addTodo(newItem)).toMatchSnapshot()
})
