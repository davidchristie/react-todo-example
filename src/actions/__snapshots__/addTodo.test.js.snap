// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
Object {
  "payload": Object {
    "completed": false,
    "text": "Test todo item test",
  },
  "type": "ADD_TODO",
}
`;
